src/cli/utils/sharepoint.go

```diff
@@ -66,6 +66,30 @@ func MakeSharePointOpts(cmd *cobra.Command) SharePointOpts {
 	}
 }
 
+func SharePointAllowedCategories() map[string]struct{} {
+	return map[string]struct{}{
+		flags.DataLibraries: {},
+		// flags.DataLists:     {},
+	}
+}
+
+func AddCategories(sel *selectors.SharePointBackup, cats []string) *selectors.SharePointBackup {
+	if len(cats) == 0 {
+		sel.Include(sel.LibraryFolders(selectors.Any()), sel.Lists(selectors.Any()))
```

Comment:
If lists isn't an allowed category (line 72), then we shouldn't include it in this selector.

In the future, when it's enabled, you should be able to use this func instead.
```suggestion
		sel.Include(sel.AllData())
```